shards/modules/debug/debug.cpp

```diff
@@ -0,0 +1,43 @@
+#include <shards/core/shared.hpp>
+#include <shards/core/params.hpp>
+#include <shards/common_types.hpp>
+
+namespace shards::Debug {
+struct DebugNoop {
```

Comment:
there is already `Assert.Break`

Reply:
Although this works differently I see! You just wanna put a breakpoint manually I guess here.

Reply:
Yeah just kind of something I can put a breakpoint on without actually breaking always.